templates/forms/22-contact.html.twig

```diff
@@ -92,7 +92,7 @@
                         <div class="govuk-grid-column-three-quarters">
                             <p class="govuk-!-font-size-24">
                                 <a href="tel:03454102222">0345 410 2222</a></p>
-                            <p>{{ cscMessage }}</p>
+                            <p></p>
```

Comment:
I think you removed this by mistake?

Reply:
Ahh thanks for spotting nice spot. I've added that back. Was testing something on the contact page.